src/targets/gpu/driver/compile_pointwise.cpp

```diff
@@ -14,8 +14,14 @@ struct compile_pointwise : action<compile_pointwise>
     {
         context ctx;
         auto inputs = p.parse_shapes(v.at("inputs"));
-        auto op     = gpu::compile_pointwise(ctx, inputs, v.at("lambda").to<std::string>());
-        double t    = time_op(ctx, op, inputs, p.get(v, "iterations", 100));
+
+        int global = p.get(v, "global", 64);
+        int local  = p.get(v, "local", 64);
```

Comment:
If global and local are not provided then it should be omitted from the function call so the defaults will get called. You can do `p.get(v, "global", 0)` and then if its equal to `0` you know it was not provided.

Reply:
I don't see how that is different than passing the default as an argument.  Can you clarify?